module/Decision/src/Decision/Service/Member.php

```diff
@@ -27,9 +28,43 @@ public function getMembershipInfo()
     }
 
     /**
+     *
+     */
+    public function findMemberByLidNr($lidnr)
+    {
+        return $this->getMemberMapper()->findByLidnr($lidnr);
+    }
+
+    /**
+     * Find a member by (part of) its name.
+     *
+     * @param string $query (part of) the full name of a member
+     * @pre $name must be at least 3 characters
+     *
+     * @return array|null
+     */
+    public function searchMembersByName($query)
+    {
+        if (strlen($query) < 3) {
+            throw new \Zend\Code\Exception\InvalidArgumentException(
+                $this->getTranslator()->translate('Name must be at least 3 characters')
```

Comment:
Why does the name need to be at least 3 characters? 1 or 2 seems much more reasonable here actually.